Application/EdFi.Ods.SandboxAdmin/Views/Application/Index.cshtml

```diff
@@ -0,0 +1,208 @@
+                        </div>
+                    </td>
+                    <td data-bind="text: Id" class="application-id" style="display: none;"></td>
+                    <td data-bind="text: ApplicationName" class="application-name"></td>
+                    <td data-bind="text: EducationOrganizationId" class="application-key"></td>
+                    <td data-bind="text: VendorName" class="operational-context-name"></td>
+                </tr>
+            </tbody>
+            <tfoot>
+                <tr>
+                    <th colspan="5" data-bind="text: applicationStatus"></th>
+                </tr>
+            </tfoot>
+        </table>
+    </fieldset>
+        </div>
+    </div>
+</div>
+<div class="modal fade" tabindex="-1" data-bind="with: addApplicationDialog, attr: { id: addApplicationDialog.htmlId }" id="modal-add">
+    <div class="modal-dialog">
+        <div class="modal-content">
+            <div class="modal-header">
+                <h3 class="modal-title">Add Application</h3>
+                <button type="button" class="close icon icon-remove" data-dismiss="modal">x</button>
+            </div>
+            <div class="modal-body">
+                <fieldset>
+                    <div class="control-group">
+                        <label class="control-label" for="txtName1">Application Name:</label>
+                        <div class="controls">
+                            <input id="txtName1" type="text" data-bind="value: applicationName, valueUpdate: 'keyup'" class="form-control" placeholder="Name of Application" />
+                        </div>
+                    </div>
+                    <div class="control-group">
+                        <label class="control-label" for="txtName2">EducationOrganizationId:</label>
+                        <div class="controls">
+                            <input id="txtName2" type="text" data-bind="value: educationOrganizationId, valueUpdate: 'keyup'" class="form-control" placeholder="EducationOrganizationId" />
+                        </div>
+                    </div>
+                    <div class="control-group">
+                        <label class="control-label" >Vendor :</label>
+                        <div class="controls">
+                            <select data-bind="options: vendorList, value:selectedVendor, optionsText: 'Name', valueUpdate: 'keyup' " class="dropdown"></select>
+                        </div>
+                    </div>
+                </fieldset>
+            </div>
+            <div class="modal-footer">
+                <div style="flex: 1">
+                    <span class="fa fa-spinner fa-spin dialog-footer-icon d-none wait-spinner"></span>
+                </div>
+                <div>
+                    <button id="btn-cancel-add" class="btn" data-dismiss="modal">Cancel</button>
+                    <button id="btn-ok-add" type="submit" class="btn btn-primary" data-bind="click: onOkClicked">Add</button>
```

Comment:
The `Add` button should only be clickable when the `Application Name`, `EducationOrganizationId` and `Vendor` are initialized. You could refer to the existing logic in the "Add Sandbox" popup.